BlakeDonn/fashion_wars
# Initial structure bd

#### What's this PR do?  

- Adds initial files for components (Homepage, Results, Skins, App)
- Adds initial styling to homepage and results
- Adds two initial fetch requests
- Adds checkboxes in homepage so a user can check which skins they are interested in
- Hook useState up with checkboxes to update state dynamically 
- Uses router to route to results page when submit button is clicked
- Adds Unit testing for Homepage and Results, Integration testing for Homepage and App

#### Where should the reviewer start? 

- Start by running the application locally and testing functionality so far

#### How should this be manually tested?  

- Run all tests and inspect code, ensure state is being updated visually with react dev tools
- Test button redirect and displays results page correctly

#### Any background context you want to provide?  

- Next step is to fetch the API data and do some filtering

#### What are the relevant tickets?  

closes #1 
closes #2 
closes #3 
closes #4
closes #5 
closes #6 
closes #7
closes #8
closes #9 
closes #10
closes #12 

#### Screenshots (if appropriate) 
![Screen Shot 2020-11-06 at 2 53 12 AM](https://user-images.githubusercontent.com/65369275/98340341-386e8c80-1fdb-11eb-8fe5-9f68eadcdf92.png)

![Screen Shot 2020-11-06 at 2 53 36 AM](https://user-images.githubusercontent.com/65369275/98340384-46241200-1fdb-11eb-8236-959ceb1f8f92.png)

![Screen Shot 2020-11-06 at 2 54 05 AM](https://user-images.githubusercontent.com/65369275/98340423-576d1e80-1fdb-11eb-8408-8a2088de7444.png)
